functions.sh

```diff
@@ -544,6 +544,34 @@ install_user_mode_emulation_binary() {
     install -m 755 -o root -g root ${EMULATOR} ${USR_BIN}
 }
 
+# Recursively installs file structure from the passed folder to the chroot
+# environment.
```

Comment:
Let's substitute `folder` for `directory` (see my comment above) and `installs` for `copies`. Also, remember about the articles.

> Recursively copies the file structure from the passed directory to the chroot environment.